src/ui/server/constants/templates/insurance/policy/index.ts

```diff
@@ -11,6 +11,7 @@ export const POLICY_TEMPLATES = {
   DIFFERENT_NAME_ON_POLICY: 'insurance/policy/different-name-on-policy.njk',
   OTHER_COMPANY_DETAILS: 'insurance/policy/other-company-details.njk',
   BROKER_DETAILS: 'insurance/policy/broker-details.njk',
+  BROKER_ADDRESSES: 'insurance/policy/broker-addresses.njk',
```

Comment:
future suggestion, we could store `insurance/policy` in a constant so if we change the folder name, only change in one place